src/main/java/com/artipie/api/ErrorMessage.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie.api;
+
+/**
+ * Error message.
+ * @since 0.26
+ */
+@FunctionalInterface
+public interface ErrorMessage {
```

Comment:
Do we really need this class?
It can be easily replaced with standard `Supplier<String>`.